.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
```

Comment:
I hadn't thought to look this up yet, so for my information: Will this run daily in forked repositories as well, or just in the PEcAnProject organization? For this PR it's not likely to matter much (issues are turned off by default in forked repos and I doubt very many people have turned them on), but it could be important to consider when designing other Actions changes in the future.

Reply:
Yes, the scheduled workflow's do run on forked repos also but we can handle it using
```
jobs:
  stale:
    if: (github.event_name == "schedule" && github.repository == "PecanProject/pecan")
```

Reply:
Cool, that's easy enough. I don't think we need it for this PR, but let's remember it for future work.